### ISSUE TITLE
Dockerfile build error

### ISSUE THREAD

**Issue by YoHimik** [mgp25/Instagram-API#2615]
I'm trying to build image from Dockerfile and get this error:
```bash
$ docker build ./ --no-cache
Sending build context to Docker daemon  2.336MB
Step 1/6 : FROM php:7.0-apache-jessie
 ---> 19729c599655
Step 2/6 : RUN echo "deb http://ftp.debian.org/debian jessie-backports main" >> /etc/apt/sources.list     && apt-get update     && apt-get install -y         libfreetype6-dev         libjpeg62-turbo-dev         libmcrypt-dev         libpng12-dev         git         ffmpeg     && docker-php-ext-install -j$(nproc) iconv mcrypt     && docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/     && docker-php-ext-install -j$(nproc) gd     && docker-php-ext-install -j$(nproc) bcmath     && docker-php-ext-install -j$(nproc) exif
 ---> Running in 128bcf1d54ae
Ign http://ftp.debian.org jessie-backports InRelease
Get:1 http://security.debian.org jessie/updates InRelease [44.9 kB]
Ign http://deb.debian.org jessie InRelease
Get:2 http://deb.debian.org jessie-updates InRelease [7340 B]
Ign http://ftp.debian.org jessie-backports Release.gpg
Ign http://ftp.debian.org jessie-backports Release
Err http://ftp.debian.org jessie-backports/main amd64 Packages
  
Get:3 http://deb.debian.org jessie Release.gpg [2420 B]
Err http://ftp.debian.org jessie-backports/main amd64 Packages
  
Err http://ftp.debian.org jessie-backports/main amd64 Packages
  
Err http://ftp.debian.org jessie-backports/main amd64 Packages
  
Get:4 http://deb.debian.org jessie Release [148 kB]
Err http://ftp.debian.org jessie-backports/main amd64 Packages
  404  Not Found
Get:5 http://security.debian.org jessie/updates/main amd64 Packages [825 kB]
Get:6 http://deb.debian.org jessie/main amd64 Packages [9098 kB]
Fetched 10.1 MB in 8s (1168 kB/s)
W: Failed to fetch http://deb.debian.org/debian/dists/jessie-updates/InRelease  Unable to find expected entry 'main/binary-amd64/Packages' in Release file (Wrong sources.list entry or malformed file)

W: Failed to fetch http://ftp.debian.org/debian/dists/jessie-backports/main/binary-amd64/Packages  404  Not Found

E: Some index files failed to download. They have been ignored, or old ones used instead.
The command '/bin/sh -c echo "deb http://ftp.debian.org/debian jessie-backports main" >> /etc/apt/sources.list     && apt-get update     && apt-get install -y         libfreetype6-dev         libjpeg62-turbo-dev         libmcrypt-dev         libpng12-dev         git         ffmpeg     && docker-php-ext-install -j$(nproc) iconv mcrypt     && docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/     && docker-php-ext-install -j$(nproc) gd     && docker-php-ext-install -j$(nproc) bcmath     && docker-php-ext-install -j$(nproc) exif' returned a non-zero code: 100
```
Can you help?

**Comment by ihatov08** [comment]
@YoHimik 
Try this code!

Dockerfile
```
FROM php:7.0-apache-jessie
RUN echo "deb http://deb.debian.org/debian jessie main" > /etc/apt/sources.list \
    && echo "deb http://security.debian.org jessie/updates main" >> /etc/apt/sources.list \
    && apt-get update \
    && apt-get install -y \
        libfreetype6-dev \
        libjpeg62-turbo-dev \
        libmcrypt-dev \
        libpng12-dev \
        git \
        libav-tools \
        unzip \
    && docker-php-ext-install -j$(nproc) iconv mcrypt \
    && docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ \
    && docker-php-ext-install -j$(nproc) gd \
    && docker-php-ext-install -j$(nproc) bcmath \
    && docker-php-ext-install -j$(nproc) exif \
    && docker-php-ext-install -j$(nproc) pdo_mysql

ENV LANG=ja_JP.UTF-8
# Install Composer and make it available in the PATH
RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/bin/ --filename=composer

# Set the WORKDIR to /app so all following commands run in /app
WORKDIR /app

COPY composer.json composer.lock ./

# Install dependencies with Composer.
RUN composer install --no-interaction

COPY . .

```

**Comment by mgp25** [comment]
Hello @YoHimik if solution provided by @ihatov08 works let us know.

If it works,  @ihatov08 you can create a PR

Regards

**Comment by JRoy** [comment]
Should be fixed in 55501d2392a00a2b75a91dbe4dd743487f2f7055

Regards
